src/stacks-hierarchy/Root_PurchaseOverviewScreen/Root_PurchaseOverviewScreen.tsx

```diff
@@ -127,17 +128,23 @@ export const Root_PurchaseOverviewScreen: React.FC<Props> = ({
   const focusRefs = useFocusRefs(params.onFocusElement);
 
   return (
-    <View style={styles.container}>
-      <FullScreenHeader
-        title={getTextForLanguage(params.fareProductTypeConfig.name, language)}
-        leftButton={{
+    <FullScreenView
+      headerProps={{
+        title: getTextForLanguage(params.fareProductTypeConfig.name, language),
+        leftButton: {
           type: 'cancel',
           onPress: closeModal,
-        }}
-        globalMessageContext={GlobalMessageContextEnum.appTicketing}
-        setFocusOnLoad={!params.onFocusElement}
```

Comment:
Should be set to `false` instead of removed, as the default is `true`.

Reply:
Doh 😵‍💫